README.md

```diff
@@ -1 +1,51 @@
 # project-liberation
+
```

Comment:
Please also add under `project-liberation` that this application is CodePoets home page.